lib/workers/global/config/parse/env.ts

```diff
@@ -237,7 +228,28 @@ export async function getConfig(
     'VSTS_TOKEN',
   ];
 
-  unsupportedEnv.forEach((val) => delete env[val]);
+  for (const val of unsupportedEnv) {
+    delete env[val];
+  }
 
   return config;
 }
+
+async function parseAndValidateOrExit(
+  env: NodeJS.ProcessEnv,
+  configEnvKey: string,
+): Promise<AllConfig> {
+  if (!env[configEnvKey]) {
+    return {};
+  }
+
+  try {
+    const config = parseJson(env[configEnvKey], '.json5') as AllConfig;
```

Comment:
```suggestion
    const config = parseJson(env[configEnvKey], `${configEnvKey}.env.json5`) as AllConfig;
```
should be more descriptive

Reply:
given that this is will be taken to the json5 trinery branch, the filename will never be printed or used. so actually i think `.json5` is cleaner. but i dont mind either :)